.github/workflows/add-issue-to-project.yml

```
@@ -0,0 +1,15 @@
+name: Add issue to project
+
+on:
+  issues:
+    types:
+      - opened
+
+jobs:
+  add-issue-to-project:
+    runs-on: ubuntu-latest
+    steps:
+      - uses: UCL-MIRSG/.github/actions/add-to-project@v0.24.0
+        with:
+          app_id: ${{ secrets.APP_ID }}
+          app_pem: ${{ secrets.APP_PEM }}
```

This got changed yesterday UCL-MIRSG/.github#40
```suggestion
          app-id: ${{ secrets.APP_ID }}
          app-pem: ${{ secrets.APP_PEM }}
```